marlowe-runtime-web/test/Spec.hs

```diff
@@ -139,7 +140,7 @@ instance Arbitrary Web.PostTransactionsRequest where
     <*> arbitrary
     <*> arbitrary
     <*> arbitrary
-    <*> arbitrary
+    <*> arbitraryNormal  -- FIXME: This should handle merkleized input, too.
```

Comment:
Shrinkage doesn't work for merkleized contracts in the webserver tests. I've asked Jamie about this, since it could be a bug or limitation in the webserver. Without this change, the property-based test never terminates.